src/dispatch/mod.rs

```diff
@@ -2,10 +2,11 @@
 pub use self::async_dispatcher::AsyncDispatcher;
 #[cfg(feature = "parallel")]
 pub use self::par_seq::{Par, ParSeq, RunWithPool, Seq};
-pub use self::{builder::DispatcherBuilder, dispatcher::Dispatcher};
+pub use self::{batch_builder::*, builder::DispatcherBuilder, dispatcher::Dispatcher};
```

Comment:
Please don't use glob import; we don't want all of the types to be exposed (for example, `UncheckedWorld` probably shouldn't be visible).